lk2nd/smp/clock-msm8953.c

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright (c) 2022, Stephan Gerhold <stephan@gerhold.net> */
+
+#include <board.h>
+#include <boot.h>
+#include <clock.h>
+#include <platform/iomap.h>
+#include <reg.h>
+
+#define MHZ				1000000
+
+static int lk2nd_8953_clock_setup(void *dtb, const char *cmdline,
+				  enum boot_type boot_type)
+{
+
+	/* Downstream sets this itself */
+	if (boot_type & (BOOT_DOWNSTREAM | BOOT_LK2ND))
+		return 0;
+
+	// select clock for secondary cluster
+	clk_get_set_enable("a53ssmux_c0", 800 * MHZ, 1);
```

Comment:
```suggestion
	clk_get_set_enable("a53ssmux_c0", 800 * MHZ, 0);
```

Is it necessary to enable the clock here? I think this should happen automatically when Linux is booted. Make sure to test that it still works correctly with this change.